src/Manager.js

```diff
@@ -0,0 +1,29 @@
+class Manager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  addTheCohort(cohort) {
+    this.cohorts.push(cohort)
+  }
+
+  deleteCohortByName(name) {
+    const targetCohort = this.getCohortByName(name)
+    if (!targetCohort) {
+      throw new Error('This cohort does not exist')
+    }
```

Comment:
This is redundant because the getCohortByName function throws an error if it can't find the cohort

Reply:
ok yeah I will check it thanks